src/main/java/org/spongepowered/common/mixin/core/world/MixinWorldServer.java

```diff
@@ -944,7 +944,6 @@ private boolean onAddBlockEvent(WorldServer.ServerBlockEventList list, Object ob
 
         final LocatableBlock locatable = LocatableBlock.builder()
             .location(new Location<>(this, pos.getX(), pos.getY(), pos.getZ()))
-            .state(this.getBlock(pos.getX(), pos.getY(), pos.getZ()))
```

Comment:
Same with here, either change all of them to use `location` or the chained `pos`, and `world` methods.